docs/sources/microsoft-365/msft-teams/msft-teams.yaml

```diff
@@ -146,7 +146,57 @@ endpoints:
     transforms:
       - !<redact>
         jsonPaths:
-          - "$..user.displayName"
+          - "$.['@odata.context']"
+      - !<redact>
+        jsonPaths:
+          - "$..emailAddress"
```

Comment:
this should be pseudonymize, right? You are redacting all emailAddress here. Better to have 2 identities to fall back just in case